### PR TITLE
Fix build warnings

### PR DESCRIPTION
**Pull request by gewarren**
Also some Acrolinx.

### CODE REVIEW

File: docs/machine-learning/tutorials/phone-calls-anomaly-detection.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ In this tutorial, you learn how to:
 > * Detect period for a time series
 > * Detect anomaly for a periodical time series
 
-You can find the source code for this tutorial at the [dotnet/samples](https://github.com/dotnet/samples/tree/master/machine-learning/tutorials/ProductSalesAnomalyDetection) repository.
+You can find the source code for this tutorial at the [dotnet/samples](https://github.com/dotnet/samples/tree/master/machine-learning/tutorials/PhoneCallsAnomalyDetection) repository.
```
Comment (gewarren) [inline docs/machine-learning/tutorials/phone-calls-anomaly-detection.md:20]: This link doesn't actually exist, but it matches the one on line 247.

**Comment by gewarren** [comment]
@luisquintanilla This fixes a build warning. Can you take a look?

**Comment by luisquintanilla** [comment]
I'll take a look

File: docs/machine-learning/tutorials/phone-calls-anomaly-detection.md (hunk 1 above)
Comment (luisquintanilla) [reply docs/machine-learning/tutorials/phone-calls-anomaly-detection.md:20]: Hmm...I thought that the samples PR was merged already. Should work now. Thanks @gewarren

**Comment by gewarren** [comment]
@luisquintanilla Could you approve it?

**Review by luisquintanilla — APPROVED** [review]

**Pull request merged**